qemu/tests/blk_commit.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+from autotest.client.shared import error
```

Comment:
but I do not need anything from avocado.core.exceptions...
I assume you mean use "from virttest import error_context" instead of "from autotest.client.shared import error"?